.github/workflows/community.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  issues:
+    types: [ opened ]
+  issue_comment:
+    types: [ created ]
+  pull_request:
+    types: [ opened ]
+  pull_request_review:
+    types: [ submitted ]
+  pull_request_review_comment:
+    types: [ created ]
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: "<h1>Looks good having you contribute to this project!</h1> Feel free to raise an <strong>Issue</strong>! Welcome to the community :nerd_face: If any questions please ask in [Discord](https://discord.com/invite/jZQs6Wu)"
+          pr-message: "<h1>Looks good having you contribute to this project!</h1> Feel free to create a <strong>Pull Request</strong>! Welcome to the community :nerd_face: If any questions please ask in [Discord](https://discord.com/invite/jZQs6Wu)"
+
+  stats:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
I see why the workflow has failed.
We forgot to checkout 
As seen here https://github.com/EddieJaoudeCommunity/gh-action-community
```yaml
      - uses: actions/checkout@v1
      - uses: EddieJaoudeCommunity/gh-action-community/src/statistics@main
```

Reply:
Also, the `uses` you put is according to [this example](https://github.com/EddieJaoudeCommunity/gh-action-community#store-community-acitivity), and on the PR we have "gh-action-community@main". This might be a reason for failing the workflow.

Reply:
Yep, it would need `checkout` and the `path` to the Action